cmd/aws-s3-csi-mounter/main.go

```diff
@@ -0,0 +1,53 @@
+// WIP: Part of https://github.com/awslabs/mountpoint-s3-csi-driver/issues/279.
+//
+// `aws-s3-csi-mounter` is the entrypoint binary running on Mountpoint Pods.
+// It is responsible for receiving mount options from the CSI Driver Node Pod,
+// and spawning a Mountpoint instance in turn.
+// It will then wait until Mountpoint process terminates (which normally happens as a result of `unmount`).
+package main
+
+import (
+	"context"
+	"flag"
+	"os"
+	"path/filepath"
+	"time"
+
+	"k8s.io/klog/v2"
+
+	"github.com/awslabs/aws-s3-csi-driver/cmd/aws-s3-csi-mounter/csimounter"
+	"github.com/awslabs/aws-s3-csi-driver/pkg/podmounter/mountoptions"
+)
+
+var mountSockPath = flag.String("mount-sock-path", "/sock/mount.sock", "Path of the Unix socket to receive mount options from.")
+var mountSockRecvTimeout = flag.Duration("mount-sock-recv-timeout", 2*time.Minute, "Timeout for receiving mount options from passed Unix socket.")
```

Comment:
2 minutes is a long time. Why this number?

Reply:
Kinda arbitrary. This is the time this component will wait until the CSI Driver Node Pod sends mount options to it. I didn't wanted to make it short and cause retry loop. I don't expect it to wait until the whole timeout duration unless we have a bug in one of our components.